runtime/doc/eval.txt

```diff
@@ -4052,7 +4055,8 @@ jobstart({argv}[, {opts}])			{Nvim} *jobstart()*
 		  - The job ID on success, which is used by |jobsend()| and
 		    |jobstop()|
 		  - 0 when the job table is full or on invalid arguments
-		  - -1 when {argv}[0] is not executable
+		  - -1 when {cmd}[0] is not executable. (Will never fail if
+		    {cmd} is a string unless 'shell' is not executable.)
```

Comment:
It seems odd to have a sentence inside parentheses, maybe remove the parens? also there should be two spaces after period.